src/backend/marsha/core/serializers.py

```diff
@@ -283,6 +283,7 @@ class Meta:  # noqa
             "show_download",
             "should_use_subtitle_as_transcript",
             "has_transcript",
+            "resolutions",
```

Comment:
This field duplicates an information that is already in the urls field. The frontend can look in the urls field to know what resolutions are available?

Reply:
To be honnest this field is a real helper in the frontend.

Reply:
ok... I still think we should avoid duplicate information but I'm ok if you think we should keep it.